xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -1113,6 +1113,8 @@ void CVideoLibrary::UpdateVideoTag(const CVariant &parameterObject, CVideoInfoTa
     details.SetTrailer(parameterObject["trailer"].asString());
   if (ParameterNotNull(parameterObject, "tagline"))
     details.SetTagLine(parameterObject["tagline"].asString());
+  if (ParameterNotNull(parameterObject, "status"))
+    details.SetStatus(parameterObject["status"].asString());
```

Comment:
This is missing an entry in methods.json:
https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/json-rpc/schema/methods.json#L1481
```json
      { "name": "status", "$ref": "Optional.String" },
```
+ API bump.